contribute-docs/legacy-docs.md

```diff
@@ -0,0 +1,47 @@
+---
+navigation_title: Contribute to legacy docs
+---
+
+# Contribute to legacy documentation
```

Comment:
this is not legacy documentation. it's actively maintained for products that are still in support, with a legacy build system. sometimes "legacy" is a sensitive term so we should avoid sending the wrong message. this feedback applies to this entire doc.

would present it as `/guide/` vs. `/docs/` instead (like we do [here](https://elastic.github.io/docs-builder/contribute/))